packages/drivers/routerlicious-driver/src/axios.cts

```diff
@@ -0,0 +1,8 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios, { AxiosRequestConfig, AxiosRequestHeaders, RawAxiosRequestHeaders } from "axios";
```

Comment:
Is `axios` import needed?
Also for any of these that are just types, qualify with `type` importing.

Reply:
axios is not required. removed. also, updated to type importing